platform/platform.hpp

```diff
@@ -165,6 +165,7 @@ class Platform
                              TFilesWithType & outFiles);
 
   static bool IsDirectoryEmpty(string const & directory);
+  static bool IsDirectory(string const & directory);
```

Comment:
Please, add a comment describing what happens in case when there're no such directory.